python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1445,23 +1469,16 @@ def _ensure_persisted_execution_plan_snapshot(
 
         return execution_plan_snapshot_id
 
-    def _log_materialization_planned_event_for_asset(
-        self,
-        dagster_run: DagsterRun,
-        asset_key: AssetKey,
-        job_name: str,
-        step: "ExecutionStepSnap",
-        output: "ExecutionStepOutputSnap",
-        asset_graph: "BaseAssetGraph[BaseAssetNode]",
-    ) -> None:
+    def _get_partitions_keys_from_tags(
```

Comment:
this diff is brutal to read. it's factoring out some code that gets the list of partitions being materialized by a run